### PR TITLE
Add zh translation

### PR DESCRIPTION
**Pull request by octogonz**
If you want to preview it, the PR branch is deployed here: https://tiktok.github.io/sparo/

@chengcyber

### CODE REVIEW

File: apps/website/i18n/zh-cn/code.json [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+  "theme.BackToTopButton.buttonAriaLabel": {
+    "message": "回到顶部",
+    "description": "The ARIA label for the back to top button"
+  },
+  "theme.blog.archive.title": {
+    "message": "历史博文",
+    "description": "The page & hero title of the blog archive page"
+  },
+  "theme.blog.archive.description": {
+    "message": "历史博文",
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/code.json:15]:
```suggestion
    "message": "过往归档的博文",
```

File: apps/website/i18n/zh-cn/code.json [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+    "description": "The page & hero description of the blog archive page"
+  },
+  "theme.blog.paginator.navAriaLabel": {
+    "message": "博文列表分页导航",
+    "description": "The ARIA label for the blog pagination"
+  },
+  "theme.blog.paginator.newerEntries": {
+    "message": "较新的博文",
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/code.json:23]:
```suggestion
    "message": "上一页",
```

File: apps/website/i18n/zh-cn/code.json [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+    "description": "The label used to navigate to the newer blog posts page (previous page)"
+  },
+  "theme.blog.paginator.olderEntries": {
+    "message": "较旧的博文",
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/code.json:27]:
```suggestion
    "message": "下一页",
```

File: apps/website/i18n/zh-cn/code.json [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+    "description": "The label used to navigate to the older blog posts page (next page)"
+  },
+  "theme.blog.post.paginator.navAriaLabel": {
+    "message": "博文分页导航",
+    "description": "The ARIA label for the blog posts pagination"
+  },
+  "theme.blog.post.paginator.newerPost": {
+    "message": "较新一篇",
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/code.json:35]:
```suggestion
    "message": "上一篇",
```

File: apps/website/i18n/zh-cn/code.json [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+    "description": "The blog post button label to navigate to the newer/previous post"
+  },
+  "theme.blog.post.paginator.olderPost": {
+    "message": "较旧一篇",
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/code.json:39]:
```suggestion
    "message": "下一篇",
```

File: apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+## 更快的克隆！
+
+Sparo 优化了 Git 操作在大型前端 monorepo 中的性能。
+
+<!-- ---------------------------------------------------------------------------- -->
+
+## 主要特点
+
+- **熟悉的界面:** `sparo` 命令行界面 (CLI) 包装器提供了 **更好的默认设置** 和 **性能建议**，而无需更改熟悉的 `git` 语法。（本地 `git` CLI 也受支持。）
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md:29]:
```suggestion
- **熟悉的界面:** `sparo` 命令行工具 (CLI) 提供了 **更好的默认设置** 和 **性能建议**，而无需更改熟悉的 `git` 语法。（本地 `git` CLI 也受支持。）
```

File: apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+- **经过验证的解决方案:** Git 提供了[许多优化大型仓库的要素](https://tiktok.github.io/sparo/pages/reference/git_optimization/); Sparo 是将这些功能智能组合的配方。
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md:30]:
```suggestion
- **经过验证的解决方案:** Git 提供了[许多优化大型仓库的基础能力](https://tiktok.github.io/sparo/pages/reference/git_optimization/); Sparo 基于这些能力提供上层的解决方案。
```

File: apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+- **简化的稀疏签出:** 使用稀疏签出[配置文件](https://tiktok.github.io/sparo/pages/guide/sparo_profiles/) 而不是复杂的 "cones" 和 globs。
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md:31]:
```suggestion
- **简化的稀疏签出:** 使用稀疏签出[配置文件](https://tiktok.github.io/sparo/pages/guide/sparo_profiles/) 来定义文件签出范围，而不是复杂的 "cones" 和 globs。
```

File: apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+   # 从 NPM 全局安装 Sparo CLI
+   npm install -g sparo
+
+   # 使用 Sparo 克隆你的仓库
+   sparo clone https://github.com/Azure/azure-sdk-for-js.git
+
+   cd azure-sdk-for-js
+   ```
+
+   > 💡 目前支持 PNPM 和 Yarn 工作区的功能计划中，但尚未实现。欢迎贡献！
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/index.md:60]:
```suggestion
   > 💡 目前支持 NPM 和 Yarn 工作区的功能计划中，但尚未实现。欢迎贡献！
```

File: apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/ci_commands/overview.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
+---
+title: 概述
+---
+
+日常开发涉及各种 Git 操作，例如在分支之间切换、从服务器获取增量更改以及浏览历史记录。相比之下，当持续集成 (CI) 流水线签出一个 Git 分支时，通常是一个更简单的操作。文件夹或整个虚拟机mirrored可能在作业完成后立即被丢弃。因此，这两种用例需要不同的 Git 优化方法。
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/ci_commands/overview.md:5]:
```suggestion
虽然日常开发涉及多种复杂的 Git 操作，例如在分支之间切换、从服务器获取增量更改以及浏览历史记录。但是相比之下，当持续集成 (CI) 流水线签出一个 Git 分支时，通常是一个更简单的操作。CI 执行任务过程中所用到的文件夹或整个虚拟机可能在作业完成后立即被丢弃。因此，这两种用例需要不同的 Git 优化方法。
```

File: apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/ci_commands/overview.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+
+Sparo 提供了一个单独的命令行工具 `sparo-ci`，专门针对 CI 流水线进行了优化。当前实现采用以下方法：
+
+- 使用 [treeless clone](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/) 而非 **blobless clone**，假设很少需要使用 Git 历史记录。
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/ci_commands/overview.md:9]:
```suggestion
- 使用 [treeless clone](https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/) 而非 **blobless clone**，因为在 CI 环境会很少需要使用完整的 Git 历史记录。
```

File: apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/ci_commands/sparo-ci_checkout.md [hunk 12]
```diff
@@ -0,0 +1,16 @@
+---
+title: sparo-ci checkout
+---
+
+```
+sparo-ci checkout
+
+用于 CI 的特殊签出命令。它目前只接受项目选择器，例如 --to 和 --from。
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/ci_commands/sparo-ci_checkout.md:8]:
```suggestion
用于 CI 场景的签出命令。它目前只接受项目选择器，例如 --to 和 --from。
```

File: apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/ci_commands/sparo-ci_clone.md [hunk 13]
```diff
@@ -0,0 +1,18 @@
+---
+title: sparo-ci clone
+---
+
+```
+sparo-ci clone <repository> [directory]
+
+位置参数:
+  repository  要克隆的远程仓库。                            [string] [required]
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/ci_commands/sparo-ci_clone.md:9]:
```suggestion
  repository  要克隆的远程仓库地址。                            [string] [required]
```

File: apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/commands/overview.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
+---
+title: 概述
+---
+
+`git` 命令行工具支持各种子命令，例如 `git clone`、`git checkout` 等。`sparo` 命令行旨在完全替代 `git`。
+
+Sparo 有四种类型的子命令：
+
+1. **mirrored子命令**，例如 `sparo branch` 和 `sparo revert`，直接调用相应的 `git` 子命令。使用mirrored子命令的目的是让 Sparo 能够提供有关可能导致性能问题的参数的建议。此外，您还可以选择配置 Sparo 以收集匿名使用数据，帮助您衡量在您的代码库中的使用体验。（收集的数据会发送到您自己的服务，其他任何人无法访问。）
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/commands/overview.md:9]:
```suggestion
1. **mirrored子命令**，例如 `sparo branch` 和 `sparo revert`，直接调用相应的 `git` 子命令。使用mirrored子命令的目的是让 Sparo 能够提供有关可能导致性能问题的参数的建议。此外，您还可以选择在 Sparo 中配置埋点数据收集，帮助您衡量在您的代码库中的使用体验。（收集的数据会发送到您自己的服务，其他任何人无法访问。）
```

File: apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/commands/sparo_auto-config.md [hunk 15]
```diff
@@ -0,0 +1,34 @@
+---
+title: sparo auto-config
+---
+
+```
+sparo auto-config
+
+自动设置优化的 Git 配置
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/commands/sparo_auto-config.md:8]:
```suggestion
自动设置推荐的 Git 配置
```

File: apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/commands/sparo_clone.md [hunk 16]
```diff
@@ -0,0 +1,22 @@
...
+title: sparo clone
+---
+
+```
+sparo clone <repository> [directory]
+
+位置参数：
+  repository  要克隆的远程仓库。                              [string] [required]
+  directory   要克隆到的新目录名称。如果未明确指定目录名称，则使用源仓库的
+              “人性化”部分（对于 /path/to/repo.gitService 使用 repo，对于
```
Comment (jzhang026) [inline apps/website/i18n/zh-cn/docusaurus-plugin-content-docs/current/pages/commands/sparo_clone.md:11]:
```suggestion
              名字部分（对于 /path/to/repo.gitService 使用 repo，对于
```

**Comment by chengcyber** [comment]
I checked in git's translation. They officially translate "sparo checkout" to "稀疏检出". Now, it's been translated to “稀疏签出”. 

Shall we follow the same translation? It occurs multiple time across all translation.

**Comment by chengcyber** [comment]
`mirrored command` is not fully translated?  It could be `镜像子命令`

**Review by jzhang026 — APPROVED** [review]

**Pull request merged**